recent_image_and_pic_downloader/src.py

```diff
@@ -0,0 +1,56 @@
+# import selenium,time & requests modules
+import time
+
+import requests
+
+from selenium import webdriver
```

Comment:
```suggestion
import time
import requests
from selenium import webdriver
```

Avoid unnecessary newlines